### DISCUSSION TITLE
【讨论】关于原神与绝区零世界观的猜想

### DISCUSSION THREAD

**Discussion by Rope354864** [share121/inter-knot#2889] [其他]
**原神与绝区零可能是同一个世界观**
原神中的七神，在使用能力时祂们的头发会发光。而绝区零中的角色在携带了对应的音擎时，使用技能也会头发也会发光。
<img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/496b41d101eec4c792e82da91fc29daf_2136519103324822738.png" alt="时光交响之章表情包17" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/fc54df9ee5d9a9fbcfc5009f744b4f19_2142689310235023086.png" alt="时光交响之章表情包16" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/3e0d3a0b4c9c3e075949f7f8010c322d_5693816029335153202.png" alt="时光交响之章表情包15" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/6bf4ba03d77560e797085eb691f40b91_4095837989954111024.png" alt="时光交响之章表情包14" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/11/17/9d6c60e73073aa3d341ea053bcb42b82_633953060120818901.png" alt="芙宁娜-好震惊！" width="75" >
![Uploading 70dacf1fee0cdbe421.jpg…]()

**Comment by chen3537** [comment]
你是赛诺吗？<img src="https://upload-bbs.mihoyo.com/upload/2022/10/19/d82121905daa51c524da502dd489697a_2564167895718228618.png" alt="赛诺-决斗" width="75" >

**Comment by luan939150334** [comment]
原神，《提瓦特》，是世界之初降临者创造的蛋壳（大概吧），里面都是原住民（土著）。魔女和1-4降临者是来自世界之外。
绝区零，零号空洞，头发发光是空洞影响的（虽然没有证据），有了空洞，人们有不同的以太适应性。
硬要说一个世界观到没必要，只能说ZZZ专武特效设计的太专了，为了美术风格，把武器圣遗物都改的不影响战斗外观。
